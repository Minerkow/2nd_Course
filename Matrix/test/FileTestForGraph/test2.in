1 -- 2, 1;
1 -- 6, 0;
2 -- 3, 2;
2 -- 7, 2;
3 -- 4, 2;
3 -- 8, 2;
4 -- 5, 1;
4 -- 9, 2;
5 -- 10, 0;
6 -- 7, 1;
6 -- 11, 0;
7 -- 8, 2;
7 -- 12, 2;
8 -- 9, 2;
8 -- 13, 2;
9 -- 10, 1;
9 -- 14, 2;
10 -- 15, 0;
11 -- 12, 1;
12 -- 13, 2;
13 -- 14, 2;
14 -- 15, 1;
11 -- 16, 0.0;
15 -- 20, 0.0;
20 -- 16, 0.0; 120V