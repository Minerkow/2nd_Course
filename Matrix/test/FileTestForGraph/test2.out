
1 -- 2: 20 A
1 -- 6: -20 A
2 -- 3: 20 A
2 -- 7: 0 A
3 -- 4: 20 A
3 -- 8: 0 A
4 -- 5: 20 A
4 -- 9: 0 A
5 -- 10: 20 A
6 -- 7: 20 A
6 -- 11: -40 A
7 -- 8: 20 A
7 -- 12: 0 A
8 -- 9: 20 A
8 -- 13: 0 A
9 -- 10: 20 A
9 -- 14: 0 A
10 -- 15: 40 A
11 -- 12: 20 A
12 -- 13: 20 A
13 -- 14: 20 A
14 -- 15: 20 A
11 -- 16: -60 A
15 -- 20: 60 A
20 -- 16: 60 A
